poetry/repositories/legacy_repository.py

```diff
@@ -419,7 +404,7 @@ def _get_release_info(self, name: str, version: str) -> dict:
 
         return data.asdict()
 
-    def _get(self, endpoint: str) -> Optional[Page]:
```

Comment:
What was wrong with _get?

Reply:
> renamed LegacyRepositoy._get() as LegacyRepository._get_page(): so far as I can see it has little in common with its parent class's PyPiRepository._get(), and mypy complains about them not being the same

Reply:
I think the correct solution here is going to be refactoring LegacyRepository to not inherit from PypiRepository. I'd suggest doing that in a follow-up PR, and leaving some `# type:` and `# FIXME` annotations here in the meantime.

Reply:
Even if that's the right long-term fix, `get_page()` seems to me a sensible name for this method: which gets a `Page`.  Indeed separating this class from the `PyPiRepository` would leave _less_ reason for their methods to have the same names.

So I'd prefer to leave this change in place: so that we can leave this file mypy-clean - rather than having to whitelist it and allow other errors to slip back in.

(I do not expect to follow up with the change that you propose.  Not that I disagree with it; but it's more work than I intend to be taking on here.)

Reply:
I think that's sounds logic -- let's go ahead and add a `# TODO:` annotation above the class as a reminder that this needs to be revisited.